modules/test/playwright/tests/object/modelBuilder.spec.ts

```diff
@@ -40,6 +40,10 @@ test('can create relationship by dragging node handles', async ({
 
 	await _objectDefinitionsPage.viewInModelBuilder();
 
+	await _modelBuilderPage.clickToggleSidebarsButton();
+
+	await _modelBuilderPage.clickFitViewButton();
```

Comment:
Hi @gabrielwas, I am not very sure about this approach. I am calling `clickFitViewButton` twice in this test and it does not look very good, but I am trying to make it resilient. This first call, together with `clickToggleSidebarsButton` ensures every thing is visible before creating relationships AND clicking the show all fields button. This fixes the bug.